src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceWorkExperienceReferences.vue

```diff
@@ -0,0 +1,291 @@
+<template>
+  <v-row>
+    <v-col v-if="mode == 'add'" md="8" lg="6" xl="4">
+      <h3 v-if="!clientId">Reference {{ newClientId }} [Up to 6]</h3>
+      <h3 v-if="clientId">Edit {{ previousFullName }}</h3>
+      <v-form ref="addWorkExperienceReferenceForm" validate-on="input" class="mt-6">
+        <v-text-field
+          v-model="lastName"
+          :rules="[Rules.required()]"
+          label="Reference Last Name"
+          variant="outlined"
+          color="primary"
+          maxlength="100"
+        ></v-text-field>
+        <v-text-field
+          v-model="firstName"
+          :rules="[Rules.required()]"
+          label="Reference First Name"
+          variant="outlined"
+          color="primary"
+          maxlength="100"
+          class="my-8"
+        ></v-text-field>
+        <v-text-field
+          v-model="email"
+          :rules="[Rules.required(), Rules.email()]"
+          label="Reference Email"
+          variant="outlined"
+          color="primary"
+          maxlength="200"
+          class="my-8"
+        ></v-text-field>
+        <v-text-field
+          v-model="phoneNumber"
+          :rules="[Rules.phoneNumber()]"
+          label="Reference Phone Number (Optional)"
+          variant="outlined"
+          color="primary"
+          maxlength="20"
+          class="my-8"
+        ></v-text-field>
+        <v-text-field
+          v-model="hours"
+          label="Work Experience Hours"
```

Comment:
I think we are missing :rules="[Rules.required()]" here.

Reply:
Good catch!